packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -330,26 +330,30 @@ class JobsTable extends React.Component {
         name: _('formNotes'),
       },
     ],
-    individualActions: [
-      {
-        handler: (job, { goTo }) =>
-          goTo({
-            pathname: '/home',
-            query: { t: 'VM', s: constructQueryString(job.vms) },
-          }),
-        disabled: job => job.type !== 'backup',
-        label: _('redirectToMatchingVms'),
-        icon: 'preview',
-      },
-      {
-        handler: (job, { goToNewTab }) => goToNewTab(`/backup/${job.id}/edit`),
-        label: _('formEdit'),
-        icon: 'edit',
-        level: 'primary',
-      },
-    ],
   }
 
+  individualActions = [
+    {
+      handler: (job, { goTo }) =>
+        goTo({
+          pathname: '/home',
+          query: { t: 'VM', s: constructQueryString(job.vms) },
+        }),
+      disabled: job => job.type !== 'backup',
+      label: _('redirectToMatchingVms'),
+      icon: 'preview',
+    },
+    {
+      handler: (job, { goTo, goToNewTab }) => {
```

Comment:
No need to move `individualActions` out of `TableProps`. `main` is already passed to the `SortedTabled`.

See: https://github.com/vatesfr/xen-orchestra/pull/5130/files#diff-125fbd6138fb6dcfdd1882014bdf15b8R397